### DISCUSSION TITLE
New Discussion Category

### DISCUSSION THREAD

**Discussion by Music47ell** [timlrx/tailwind-nextjs-starter-blog#130] [Q&A] [answered]
I think for giscus, you should create a new `Discussion Category` and use the `Title: Comments`, and select the `Discussion Format: Announcement`.

This way comments would have its own category that is separate from actual announcements. 

Also, in the https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/components/comments/Giscus.js you removed the following line:

```js
script.setAttribute('data-category', siteMetadata.comment.giscusConfig.category)
```

But if you look at the giscus website, this is included in the final script. 

```js
<script src="https://giscus.app/client.js"
        data-repo=""
        data-repo-id=""
        data-category=""  <-------
        data-category-id=""
        data-mapping="pathname"
        data-reactions-enabled="1"
        data-emit-metadata="0"
        data-theme="dark"
        crossorigin="anonymous"
        async>
</script>
```

**Comment by timlrx** [comment, marked as answer]
Ah got it. I was slightly misled by your instructions and copied the categories array from https://giscus.app/api/discussions/categories?repo={username}%2F{repoName} which results in it not working. I also did not notice a `data-category` field in https://giscus.app/ script. It seems that once you add a custom comments category, that option will come up. Let me modify the settings...

**Reply by Music47ell** [reply]
Yeah, I should've made my instructions more clear.

**Reply by timlrx** [reply]
Thanks for pointing it out, should be fixed now!

**Reply by Music47ell** [reply]
You're welcome!

**Comment by timlrx** [comment]
I think we are good with this feature! @frontendwizard would appreciate it if you could update the typescript template accordingly 🙏
